### ISSUE TITLE
subdomain cert is conflicting the main domain

### ISSUE THREAD

**Issue by renatofrota** [WordOps/WordOps#191]
Hello,

I have installed WO, created the first site as **subdomain.example.com** (so Dashboard is located at subdomain.example.com:22222 and SSL secured).

Trying to add a new site **example.com** with SSL, it fails. Being more specific, WO said an SSL certificate already exists for example.com (what is not correct) and I am given 2 options:

- use the certificate found; or
- issue a new cert

I can't use the existing one - it is actually for the subdomain.example.com - and I can't replace it (option 2 throws an error).

System Information

```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic
```
`WordOps v3.10.1`

[wordops.txt](https://transfer.vtbox.net/tQlIs/wordops.txt)

Thank you.

**Comment by renatofrota** [comment]
I am surprised no one spotted this one before as I assume it's the most common case, set "server.example.com" as hostname and add it as the 1st wo site - and add example.com wo site in sequence.

Is there something I can do to have these working? Using another linux distro maybe?

**Comment by VirtuBox** [comment]
Hello @renatofrota,
I will publish a fix asap

**Comment by renatofrota** [comment]
Ha!! ok. Not putting pressure. I was just wondering if it this could be specific to something in my scenario. However, since you are working on a fix, makes total sense to me I wait.

I appreciate your fast response and efforts. I will find a way to give back to the project.

**Comment by VirtuBox** [comment]
Issue has been fixed in the last release (v3.10.2).

Please update WordOps with the command `wo update`
